notebooks/community/sdk/SDK_End_to_End_Tabular_Custom_Training.ipynb

```diff
@@ -23,6 +23,34 @@
         "# limitations under the License."
```

Comment:
<p>Remove -&gt; Note: you may incur charges for training, prediction, storage or usage of other GCP products in connection with testing this SDK. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>